packages/ckeditor5-dev-changelog/src/utils/removechangelogentryfiles.ts

```diff
@@ -15,9 +17,16 @@ import type { ChangesetPathsWithGithubUrl } from '../types.js';
 export async function removeChangelogEntryFiles( entryPaths: Array<ChangesetPathsWithGithubUrl> ): Promise<void> {
 	logInfo( `○ ${ chalk.cyan( 'Removing the changeset files...' ) }` );
 
-	await Promise.all(
-		entryPaths
-			.flatMap( repo => repo.filePaths )
-			.map( file => fs.unlink( file ) )
-	);
+	for ( const repo of entryPaths ) {
+		const changesetDirectory = upath.join( repo.cwd, CHANGESET_DIRECTORY );
+		const entries = await readdir( changesetDirectory, { withFileTypes: true } );
```

Comment:
Why do we read anything from `fs` if we have `entryPaths`?

Reply:
I added it to detect and remove directories, but I rolled it back to the original implementation, and directories are ignored. Fixed in [e4f1e73](https://github.com/ckeditor/ckeditor5-dev/pull/1183/commits/e4f1e73009f8cbfb20a274d322f2225d0c5bcd46).